src/Metadata/Property/SubresourceMetadata.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Metadata\Property;
+
+/**
+ * Subresource metadata.
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ */
+final class SubresourceMetadata
```

Comment:
I'm -1 for this. It duplicates information already available from the `Type` via `PropertyMetadata`.